0020-day-two/stand-ups/readme.md

```diff
@@ -0,0 +1,99 @@
+# Stand-ups
+
+The stand-up is a ceremony that is held each day of the sprint. In this ceremony, each contributor will answer three simple questions. This will repeat until each contributor has answered the three following questions.
+
+1. What did you work on yesterday?
+2. What are you working on today?
+3. Do you have any impediments/blockers?
+
+After that point, the stand-up is concluded. There may be a parking lot discussion after. However, participation in the parking lot discussion is optional for all members except those explicitly needed for discussion of the issues raised.
+
+The term parking lot refers to a bucket of comments, concerns, or questions that will be discussed and/or addressed at a later point with potentially fewer contributors. This is part of a strategy to avoid letting the discussion in a meeting shift to a subject that is not aligned with the meeting goals and/or decisions.
+
+## Goals
+
+1. Bring awareness to the contribution of each member for the past and upcoming day.
+2. Surface any impediments to one or more team members' contributions.
+
+## Participation
+
+The entire team should attend the stand-up. Anyone that worked on a task towards the sprint work should answer the three questions. It would be up to the team to decide if they would like updates from members that are not directly working against sprint task work (i.e. Product Owners and Program Managers).
+
+- Scrum Master (Required)
+- Product Owner (Optional)
+- Program Manager (Required)
+- Dev Manager (Required)
+- Tech Lead + Contributors (Required)
+
+## Impact
+
+Team members get a clear understanding of what development tasks are going on within the team which helps with collaboration. It also provides a time to address any challenges or blockers that may be stopping specific tasks from being completed, therefore helping with velocity.
+
+## Measures
+
+Both stand-up length and time to start are important as the stand-up has to be seen as a reliable and efficient meeting that facilitates communication of information versus unnecessary overhead.
+
+### Stand-up Length
+
+While the length can depend on the team size, if everyone is sticking to one line answers to the 3 key questions, it should be fairly easy to conclude within 5-10mins.
+
+- 1-5 min = Excellent
+- 5-10 min = Great
+- 10-15 min = Good
+- 15+ min = Needs Improvement

```

Comment:
This is also a function of the size of the team.  If you have a 4 person team, a 10 min standup isn't that great.  If you have a 10 person team , then a 15-min standup is pretty darned good.    Add a caveat/clarification here about team size.

Reply:
Fixed in https://github.com/microsoft/code-with-engineering-playbook/pull/67/commits/32de5bff4841f3cff550a7948fed94f9b79378cd.